pkg/bgpv1/manager/reconcilerv2/service.go

```diff
@@ -850,6 +871,77 @@ func checkServiceAdvertisement(advert v2alpha1.BGPAdvertisement, advertServiceTy
 	return true, nil
 }
 
+func mergeRoutePolicies(policyA *types.RoutePolicy, policyB *types.RoutePolicy) (*types.RoutePolicy, error) {
```

Comment:
@harsimran-pabla From a style perspective, do you prefer this to be standalone or as a member of the Reconciler instance:
```
func (r *ServiceReconciler) mergeRoutePolicies(...) {
```

Reply:
I think it can be standalone, also moved in route_policy.go file, so if required we can reuse it for merging route policies in other reconcilers.